src/wwwroot/partials/settings/billing.html

```diff
@@ -108,9 +108,9 @@ <h4 class="special plan-price">{{vm.currentCurrency}} {{vm.planPrice | number:2
           <div class="flex flex--vertically-center">
             <label for="creditCardRadio">{{'credit_card_option' | translate}}</label>
           </div>
-          <div class="flex flex--vertically-center flex--wrap">
+          <!-- <div class="flex flex--vertically-center flex--wrap">
```

Comment:
I will add a comment, I hidden radio button of payment method
![2017-06-16_1109](https://user-images.githubusercontent.com/6796523/27230077-658db182-5284-11e7-8407-51931fd97e29.png)